contracts/feeds/PriceFeedsMoC.sol

```diff
@@ -47,8 +50,9 @@ contract PriceFeedsMoC is IPriceFeedsExt, Ownable {
         if (hasValue) {
             return uint256(value);
         } else {
-            (uint256 price, ) = IRSKOracle(rskOracleAddress).getPricing();
-            return price;
+            (uint256 price, bool success) = IPriceFeedLatestAnswer(fallbackOracleAddress)
+                .latestAnswer();
+            return success ? price : 0;
```

Comment:
Returning `0` instead of reverting on failure seems very dangerous to me. If the calling code doesn't handle `0` correctly, it might do all kinds of wrong things, like enable liquidations of loans.

Reply:
@koirikivi  it's fine, this is the new deployed contract, and will only be used by the core PriceFeeds.
In the core PriceFeeds' itself will revert the 0 result price  https://github.com/DistributedCollective/Sovryn-smart-contracts/blob/120f0bc493eef5e6ab24327515bd068c87943c2c/contracts/feeds/PriceFeeds.sol#L436

Reply:
With this info, and after talking to Tyrone, I'll approve this.